test/parallel/test-buffer-tojson.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const Buffer = require('buffer').Buffer;
```

Comment:
I think I got this by copying the boilerplate from `test-buffer-new`..ah, that one doesn't need this too :/.